### PR TITLE
Make it possible to add angular components to the "CSS library"

### PR DESCRIPTION
**Pull request by proand**
Both ngBootstrap- and fhiAngular-components can be added.
ngbAccordion is used as test case for the implementation.
Some other refactorings are made as well.

### CODE REVIEW

**Comment by proand** [comment]
There are some naming inconsistencies which we need to discuss...

File: src/mock-db-data/library-css/components/accordion.ts [hunk 1]
```diff
@@ -1,138 +1,46 @@
...
+  <ngb-panel id="custom-id-2" title="Item #3">
+    <ng-template ngbPanelContent>
+      <p>Item #3 content</p>
+    </ng-template>
+  </ngb-panel>
+</ngb-accordion>`,
+  documentationHtml: `
+<p>
+  Bootstrap documentation for
+  <a href="${BootstrapBaseUrl}/${itemTitle.toLocaleLowerCase}">Accordion</a>
```
Comment (yleaxeman) [inline src/mock-db-data/library-css/components/accordion.ts:35]: Actual URL generated: https://getbootstrap.com/docs/5.2/components/function%20toLocaleLowerCase()%20%7B%20%20%20%20[native%20code]%7D

Comment (yleaxeman) [reply src/mock-db-data/library-css/components/accordion.ts:35]: Missing parentheses on toLocaleLowerCase method

**Review by yleaxeman — CHANGES_REQUESTED** [review]

File: src/mock-db-data/library-css/components/accordion.ts (hunk 1 above)
Comment (proand) [reply src/mock-db-data/library-css/components/accordion.ts:35]: Ups... the dev-server shouldn't let let that pass me think 😉

File: src/mock-db-data/library-css/components/accordion.ts [hunk 2]
```diff
@@ -1,138 +1,46 @@
...
+  </ngb-panel>
+</ngb-accordion>`,
+  documentationHtml: `
+<p>
+  Bootstrap documentation for
+  <a href="${BootstrapBaseUrl}/${itemTitle.toLocaleLowerCase()}">Accordion</a>
+</p>
+<p>
+  The accordion is implemented as a
+  <a href="${NgBootstrapBaseUrl}/${itemTitle.toLocaleLowerCase}">ngBootstrap Accordion</a>
```
Comment (yleaxeman) [inline src/mock-db-data/library-css/components/accordion.ts:39]: Same issue as above. I forgot to point that out.

**Review by yleaxeman — CHANGES_REQUESTED** [review]

File: src/mock-db-data/library-css/components/accordion.ts (hunk 2 above)
Comment (proand) [reply src/mock-db-data/library-css/components/accordion.ts:39]: Hehe, my bad, had to run to meeting, so didn't look before jumping 😉

**Review by yleaxeman — APPROVED** [review]

**Pull request merged**